boards/arduino-mega2560/include/periph_conf.h

```diff
@@ -264,23 +265,56 @@ extern "C" {
 /**
  * @brief GPIO configuration
  */
-#define GPIO_NUMOF          (0U)
-#define GPIO_0_EN           0
-#define GPIO_1_EN           0
-#define GPIO_2_EN           0
-#define GPIO_3_EN           0
-#define GPIO_4_EN           0
-#define GPIO_5_EN           0
-#define GPIO_6_EN           0
-#define GPIO_7_EN           0
-#define GPIO_8_EN           0
-#define GPIO_9_EN           0
-#define GPIO_10_EN          0
-#define GPIO_11_EN          0
-#define GPIO_12_EN          0
-#define GPIO_13_EN          0
-#define GPIO_14_EN          0
-#define GPIO_15_EN          0
+#define GPIO_NUMOF          7
+#define GPIO_0_EN           1
+#define GPIO_1_EN           1
+#define GPIO_2_EN           1
+#define GPIO_3_EN           1
+#define GPIO_4_EN           1
+#define GPIO_5_EN           1
+#define GPIO_6_EN           1
+
+/* GPIO channel 0 configuration */
+#define GPIO_0_PORT         PORTA
+#define GPIO_0_PIN          PA0
+#define GPIO_0_DDR          DDRA
+#define GPIO_0_STATE        PINA
```

Comment:
I don't like the name of `GPIO_0_STATE ` but neither I have a better idea atm. It's just because the other defines are somehow related to the actual register names but "state" is not. Hm..